### PR TITLE
Split OMERO virtual appliance downloads page and generation script

### PR DESCRIPTION
**Pull request by sbesson**
- Move the OMERO Virtual Appliance to a separated downloads page
- Remove references to the OMERO Virtual appliance

### CODE REVIEW

File: omerova_downloads.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                </div>
+                <ul>
+                    <li>
+                        <p>The OMERO Virtual Appliance allows you to try out an
+                            OMERO server on your local machine </p>
+                    </li>
+                    <li>
+                        <p>Instructions for using the Virtual Appliance are at:
+                                <a href="@DOC_URL@/users/virtual-appliance.html"
+                                >Virtual Appliance</a></p>
```
Comment (hflynn) [inline omerova_downloads.html]: Virtual Appliance User Guide?

**Comment by hflynn** [comment]
Or actually, "See Virtual Appliance user guide for installation instructions"

**Comment by hflynn** [comment]
Otherwise looks good!

**Comment by hflynn** [comment]
:+1:

**Comment by hflynn** [comment]
Ok, only remaining issue I have is that the 'previous versions' section is showing the Virtual Appliance download folder so nothing earlier than 5.1.2. We probably need to provide two links here:

"We recommend that you always use the most recent release version of the OMERO software, but if you need to use a previous version, they are available from the Virtual Appliance downloads folder (for versions from 5.1.2 onwards) or the main OMERO downloads folder (versions prior to 5.1.2 are provided on the main OMERO downloads pages).

**Comment by manics** [comment]
I'm tempted to say don't bother with that caveat- it's purely for testing OMERO so there's no good reason for someone unfamiliar with the project to want an older verison.

**Comment by hflynn** [comment]
Ah good point, you could just remove the entire previous versions section.

**Comment by sbesson** [comment]
Pushed and downloads page regenerated.

**Comment by hflynn** [comment]
Good to merge

**Pull request merged**